### DISCUSSION TITLE
List connected clients under systray icon with option to "disconnect"

### DISCUSSION THREAD

**Discussion by KuleRucket** [LizardByte/community#250] [Sunshine]
This would allow you to see what is connected and allow you to kick clients. It is also a useful way to exist the stream if you only have a mouse handy.

**Comment by ReenigneArcher** [comment]
I like this idea! I would also like to enable optional notifications when a stream starts or ends... including the reason for termination.

**Comment by skoshy** [comment]
Is there anyway to do this via the command line at least for now, or the web interface?

I sometimes connect to my PC from my laptop and forget to disconnect, meaning it'll either (1) not connect from my phone if channels=1 or (2) will duplicate the stream / waste resources if I connect from my phone.

**Reply by ReenigneArcher** [reply]
Not right now
